### DISCUSSION TITLE
Purge recordings by date?

### DISCUSSION THREAD

**Discussion by nickds1** [mcguirepr89/BirdNET-Pi#1162] [General]
Hi - Love Birdnet-PI, now running over 6 months on an 8GB Pi4B

...which means I now have 19G in /home/pi/BirdSongs/Extracted and my 32GB uSD card is nearly full ... I know about the "purge" setting, and it is enabled, it's just that I need a bit of space for other stuff...

I don't really want all those recordings  - 220,000 of them - the last couple of months would be nice.  Is there anything I can do to cut off, say all of 2023, but leave the stats. Or even lose the 2023 stats too? I can provide an external link to the Pi (it has an external URL).

Thanks for a great project.

Nick

**Comment by MueJosh** [comment]
Hey, 

You can delete them by selecting the year. The easy way is to log in via SSH and then deleting them like any other file / folder. 
`USER/BirdSongs/Extracted/By_Date` 

You could also backup the data (for example on windows with winscp or similar Apps)

The stats are still there (just DONT delete the `birds.db` file)

I hope this helps, if not- let us know.

**Comment by hpk60** [comment]
As my question concerns the purging as well, I post it here.

Can someone tell my, what happens, when I lock/unlock a detection on the recordings page. 
Does it alter the file permissions, so that a file is not purged or does it add an info in the birds.db?
